<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lighting</class>
 <widget class="QWidget" name="Lighting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_panel_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>21</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(243, 249, 255, 10);
}</string>
   </property>
   <property name="text">
    <string>   Lighting</string>
   </property>
  </widget>
  <widget class="Line" name="separator_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_switcher">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_switcher">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}

QComboBox::disabled {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(100, 100, 100, 0.7);
	border: 1px solid rgba(100, 100, 100, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}</string>
   </property>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ON</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_light_rgb">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>233</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R: 100; G: 100; B: 100;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="palette_light">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgba(90, 90, 90, 0.6);
	border-radius: 13px;
	background-color: rgba(100, 100, 100, 1.0);
	color: rgba(0, 0, 0, 0);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #404040 stop: 1 #686868);
}</string>
   </property>
   <property name="text">
    <string>â¬¤</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_light_color">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>203</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="d_spin_box_y_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>343</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>QDoubleSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QDoubleSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-100000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>263</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X position</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="d_spin_box_x_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>283</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QDoubleSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-100000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_z_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>383</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z position</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="d_spin_box_z_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>403</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>QDoubleSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QDoubleSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-100000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>323</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y position</string>
   </property>
  </widget>
  <widget class="Line" name="separator_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>433</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_power">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.Arabic UI Text</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="d_spin_box_power">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QDoubleSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="separator_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
